cpp/src/slice2cs/Gen.cpp

```diff
@@ -2184,6 +2294,11 @@ Slice::Gen::ServantVisitor::visitInterfaceDefStart(const InterfaceDefPtr& p)
     _out << sp;
     emitAttributes(p);
 
+    ostringstream notes;
+    notes << "Your servant class implements this interface by deriving from a generated Disp_ class such as "
+          << "<see cref=\"" << p->mappedName() << "Disp_\" />.";
```

Comment:
Do we need "such as" here?
`p->mappedName() + "Disp_"` just _is_ the Disp right?
It's not like it's just one of many possible choices the user can implement.
```suggestion
    notes << "Your servant class implements this interface by deriving from the generated Disp_ class "
          << "<see cref=\"" << p->mappedName() << "Disp_\" />.";
```

Reply:
> It's not like it's just one of many possible choices the user can implement.

It is actually like that. For Slice interface Greeter, you can use any Disp_ that implements this interface, not just GreeterDisp_.